README.md

```diff
@@ -213,6 +219,8 @@ GLOBAL OPTIONS:
    --remote value, -r value  Name of the remote (default: "scalingo")
    --region value            Name of the region to use
    --version, -v             print the version
+
+Fail to run command: flag: help requested
```

Comment:
```suggestion
```

Haha not needed. It's because you type `scalingo --help` whereas the command is `scalingo help`

Reply:
haha spotted